r/tests/testthat/test-feather.R

```diff
@@ -196,3 +196,17 @@ test_that("Character vectors > 2GB can write to feather", {
 })
 
 unlink(feather_file)
+
+ft_file <- test_path("golden-files/data-arrow_2.0.0_lz4.feather")
+
```

Comment:
There should at least be a test for all lines of code you added. I don't see any parquet tests.

Also, you might want to test what happens if you pass an invalid file (like, what if you did `read_feather(parquet_file)`?)

Reply:
thanks, I'll re-add those since I thought it was a "duplication"

Reply:
Not duplication, your code changes in parquet.R are not tested without them

Reply:
In chatting with Neal about this, I had originally commented on the parquet test you added in the first PR that it wasn't being triggered. It wasn't ever triggered on the crossbow job that you were testing on that PR, but it turns out that that branch will be triggered in a different crossbow test that I had forgotten about: https://github.com/apache/arrow/blob/master/dev/tasks/tasks.yml#L1609-L1613. SO you'll want to run both of those crossbow builds when you're testing this to ensure that you're getting the messages you expect.